compiler/rustc_hir_analysis/src/collect/item_bounds.rs

```diff
@@ -334,6 +341,13 @@ fn opaque_type_bounds<'tcx>(
             | PredicateFilter::SelfTraitThatDefines(_)
             | PredicateFilter::SelfAndAssociatedTypeBounds => {
                 icx.lowerer().add_default_traits(&mut bounds, item_ty, hir_bounds, None, span);
+                icx.lowerer().adjust_sizedness_params_and_assoc_types(
```

Comment:
Same thing, reorder to keep it the same everywhere

Reply:
Fixed